src/Batch/CsvImportBatch.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Drupal\csvimport\Batch;
+
+// @codingStandardsIgnoreStart
+use Drupal\node\Entity\Node;
```

Comment:
Wouldn't hurt to explain why we import this here (for the commented code creating nodes later).

Reply:
Absolutely! Even more code documentation incoming with the next PR.